boards/laird_connect/bl654_dvk/bl654_dvk_nrf52840_pa_defconfig

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
```

Comment:
2x empty line can be 1x empty line

Reply:
I based the new defconfig on the existing one which had the extra line. I can fix it too, if you'd like.